File: chess_cpp/beans/KingIsUnderAttackResponse.cpp
//
// Created by Hrach Khachatryan on 12/15/19.
//

#include "KingIsUnderAttackResponse.h"
KingIsUnderAttackResponse::KingIsUnderAttackResponse(bool underAttack, std::string message) {
    this->underAttack = underAttack;
    this->message = message;
}

bool KingIsUnderAttackResponse::isUnderAttack() {
    return underAttack;
}

void KingIsUnderAttackResponse::setUnderAttack(bool underAttack) {
    this->underAttack = underAttack;
}

std::string KingIsUnderAttackResponse::getMessage() {
    return message;
}

void KingIsUnderAttackResponse::setMessage(std::string message) {
    this->message = message;
}

File: chess_java/src/chess/beans/KingIsUnderAttackResponse.java
package chess.beans;

public class KingIsUnderAttackResponse {
    private boolean underAttack;
    private String message;

    public KingIsUnderAttackResponse(boolean underAttack, String message) {
        this.underAttack = underAttack;
        this.message = message;
    }

    public boolean isUnderAttack() {
        return underAttack;
    }

    public void setUnderAttack(boolean underAttack) {
        this.underAttack = underAttack;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
